### DISCUSSION TITLE
Actividad 0 - Consulta

### DISCUSSION THREAD

**Discussion by dalmamunoz** [frlp-utn-ingsoft/catedra#26] [Q&A] [answered]
Buenas
Quería hacer una consulta porque estoy teniendo este error en la parte 1 y no me doy cuenta como arreglarlo. Estuve buscando en foros y no me funciona. Reviste que Python estuviera instalado, el path , Django también esta instalado. Entiendo que me pide crear un entorno virtual para el proyecto y si esta todo instalado de forma global debería funcionar de todas formas ?

Gracias, saludoss.

![WhatsApp Image 2024-04-12 at 01 53 53](https://github.com/frlp-utn-ingsoft/catedra/assets/84876749/a5ce545e-68d1-4304-891b-2ee75fba7db7)

**Comment by RodrigoJacznik** [comment, marked as answer]
Buenas, mandame una captura de cuando haces `pip install Django`.
Si estas en linux puede ser que tengas que crear un virtualenv.
Los pasos son los siguientes

1. Entrar a la carpeta del projecto
2. Crear un venv. `python -m venv .env`
3. Activar el venv (esto lo tenes que hacer cada vez que queres usar el proyecto). `source .env/bin/activate`
4. Instalar django (esto 1 sola vez). `pip install Django`
5. Cuando terminas de trabajar en el proyecto desactivar el venv. `deactivate`

**Reply by dalmamunoz** [reply]
Volví a reinstalar todo nuevamente y funciono. Estoy usando Windows, solo que estaba usando la terminal de bash en visualcode por lo que vi  eso no es motivo de error igual ya pude avanzar. Graciass !!
